cibuildwheel/platforms/android.py

```diff
@@ -580,15 +593,26 @@ def test_wheel(state: BuildState, wheel: Path) -> None:
             env=state.build_env,
         )
 
+    platform_args = (
+        [
+            "--python-platform",
+            "x86_64-linux-android" if state.config.arch == "x86_64" else "aarch64-linux-android",
```

Comment:
Minor comment:
This seems prone to errors in the future if we can't rely on `sysconfig_print`.
Maybe move this to a map so that future platforms error out rather than default to `aarch64-linux-android` ?